quorabase/models.py

```diff
@@ -0,0 +1,28 @@
+from django.db import models
+from django.contrib.auth.models import User
+from django.utils import timezone
+from vote.models import VoteModel
```

Comment:
Where is the `vote` app installed.
Can we achieve voting functionality using many to many relation and a intermediate table.

Reply:
Yes, we can achieve voting functionality by using an intermediate table. I haven't reached the voting part of my app yet, so I haven't updated it.I found this VoteModel, which makes it easy to implement a voting system.